cg-spec/editors_draft.html

```diff
@@ -462,7 +462,53 @@ <h2>Discussion</h2>
     </section>
   </section>
 
-    <section id="n-triples-star">
+  <section id="trig-star">
+    <h2>TriG-star</h2>
+
+    <p>This section describes TriG-star,
+      a minimal extension of the TriG format [[TRIG]]
+      using the same production updates described in <a href="#turtle-star" class="sectionRef"></a>.</p>
+
+    <p class="note">As RDF-star describes <a>embedded triples</a>,
+      which are not necessarily described within any specific graph,
+      either named or default.</p>
+
+    <p>A <dfn>TriG-star document</dfn> defines an <a>RDF-star dataset</a> composed of
+      a single <a>default graph</a>, and zero or more <a>named graphs</a>,
+      all of which are <a>RDF-star graphs</a>.</p>
+
+    <p>The TriG-star grammar contains exactly the same productions updates
+      described in <a href="#turtle-star-grammar" class="sectionRef"></a>.</p>
+
+    <p>TriG-star parsing uses the same updates described in <a href="#turtle-star-parsing" class="sectionRef"></a> as applied to <a data-cite="TRIG#sec-parsing">Section 5 of the TriG specification</a> [[TRIG]].</p>
+
+    <p class="note">As with Turtle-star, the <a href="#grammar-production-embTriple">`embTriple`</a> and annotation <a href="#grammar-production-annotation">`annotation`</a> yield <a>RDF-star triples</a> which are not directly added to `curGraph`.</p>
+    <p>A conforming TriG-star parser MUST parse any
+      valid <strong>TriG document</strong> and any
+      valid <a>Turtle-star document</a> in addition
+      to the <a href="#turtle-star-grammar">Turtle-star grammar</a> productions contained within a <a>named graph</a>.</p>
+
+    <section id="trig-star-discussion" class="informative">
+      <h3>Discussion</h3>
+
+      <p>TriG-star allows the same expresivity of Turtle-star with the addition of allowing <a>embedded triples</a> and <a href="#grammar-production-annotation">annotations</a> within named graphs.</p>
+
+      <pre class="example"
+           data-transform="updateExample"
+           data-content-type="text/turtle"
+           title="TriG-star annotated triples">
+      <!--
+        @base <http://example.org/> .
+        @prefix : <#> .
+        :G {
+          _:a :name "Alice" {| :statedBy :bob |} .
+        }
```

Comment:
`<pre>` contents should not be indented except where that indentation is to help human parsing (as with the single indented line in my suggestion).
```suggestion
@base <http://example.org/> .
@prefix : <#> .
:G {
  _:a :name "Alice" {| :statedBy :bob |} .
}
```

Reply:
> `<pre>` contents should not be indented except where that indentation is to help human parsing (as with the single indented line in my suggestion).

Reaped has code to remove leading white space from pre-blocks specifically to address this issue, so,it should not nave necessary to left-justify examples.

Reply:
Well, alrighty, then! :-)